sound/smooth_swing_v2.h

```diff
@@ -26,7 +26,16 @@ class SmoothSwingV2 : public SaberBasePassThrough {
       accent_swings_present = true;
     } else {
       accent_swings_present = false;
-      STDOUT.print("Accent Swings NOT Detected: ");
+      STDOUT.println("Accent Swings NOT Detected: ");
+    }
+    if (slsh.files_found() > 0 && smooth_swing_config.AccentSlashAccelerationThreshold > 0.0) {
```

Comment:
I think we should actually default slashes to be on if present AND accent swings are enabled.
I think that might mean that we don't need to have separate variables for smooth swing and non-smoothswing fonts.

Reply:
I think we need to have separate settings for the same reason we have separate settings for swing and accent swings: hybrid fonts tend to have different threshold requirements than non-hybrid

Reply:
I don't know if it is strictly needed, but it's probably going to confuse users more if we don't separate them, so let's go with that, at least for now.